File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// CorporateBullshitGenerator SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class CorporateBullshitGeneratorMakeContext
{
    public static function call(array $ctxmap, ?CorporateBullshitGeneratorContext $basectx): CorporateBullshitGeneratorContext
    {
        return new CorporateBullshitGeneratorContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# CorporateBullshitGenerator SDK utility: make_context
require_relative '../core/context'
module CorporateBullshitGeneratorUtilities
  MakeContext = ->(ctxmap, basectx) {
    CorporateBullshitGeneratorContext.new(ctxmap, basectx)
  }
end
